### PR TITLE
Add ability to override default DH_PARAM_SIZE of 2048

### PR DESCRIPTION
**Pull request by XPLOT1ON**
I would like to add the ability to change the default Diffie-Hellman parameters of 2048 bits through the environment variable. This would allow DH Param to be future proof where the user would be able to pick a desired value such as `3072` or `4096` bits

### CODE REVIEW

**Comment by JrCs** [comment]
@XPLOT1ON  what is your use case to need to change the number of bits for DH keys. It's already 2048 and give us a A+ grade on Qualsys SSL Server Test ?

**Comment by XPLOT1ON** [comment]
@JrCs Yes, 2048 does give A+ SSL raitings, but when I generate DH params, I would prefer for a bigger bit size such as 4096. and as mentioned before, if 2048 bit size is cracked (well, not very soon, I hope) a change to the DH param bit size can override the default value as needed.

File: Dockerfile [hunk 1]
```diff
@@ -4,7 +4,8 @@ MAINTAINER Yves Blusseau <90z7oey02@sneakemail.com> (@blusseau)
 
 ENV DEBUG=false              \
 	DOCKER_GEN_VERSION=0.7.3 \
-	DOCKER_HOST=unix:///var/run/docker.sock
+	DOCKER_HOST=unix:///var/run/docker.sock \
+	DH_PARAM_SIZE=2048
```
[the remark this answers is not in the document]
Comment (XPLOT1ON) [reply Dockerfile]: Isn't `ENV DH_PARAM_SIZE` would be a default value for docker environment? User can override the bits size via compose or docker run.

[the remark this answers is not in the document]
Comment (JrCs) [reply Dockerfile]: I prefer not to depend of the content of the dockerfile.
Closer to the code is better.

**Comment by buchdag** [comment]
@XPLOT1ON > let me know if you need help to complete this :)

**Comment by XPLOT1ON** [comment]
@buchdag sure. I've got sidetracked from completing this due to other commitment.

**Comment by buchdag** [comment]
@JrCs I think this is good to go now, awaiting your approval.

**Pull request merged**